config.js

```diff
@@ -29,7 +29,7 @@ var config = {};
  * Default log level. Can be one of: 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'
  * @type {string}
  */
-config.logLevel = 'DEBUG';
+config.logLevel = 'INFO';
```

Comment:
The logs.md file should be adjusted to include the new default.

In addition, I think it would be a good idea to include `config.logLevel` in configuration.md, for the sake of completeness.